### ISSUE TITLE
Property 'domain' not implemented to RegistroBR parser

### ISSUE THREAD

**Issue by anacarolinacastro** [weppos/whois-parser#76]
The property 'domain' is not implemented on RegistroBR parser.

```
> record = Whois.whois("casarobertomarinho.org.br")
> parser = Whois::Parser.new(record)
> parser.domain
Whois::AttributeNotImplemented: Unable to find a parser for property `domain'
```

**Comment by weppos** [comment]
Correct, it's not.
https://github.com/weppos/whois-parser/blob/master/lib/whois/parsers/whois.registro.br.rb

Feel free to provide a PR. Actually, the domain property has little sense to me as it's basically whatever query you passed to the whois argument.

Any reason for needing it?

**Comment by anacarolinacastro** [comment]
My commit fixes it:

https://github.com/anacarolinacastro/whois-parser/commit/03042612ac25792f7c8a3b1bb6241ebcf809c11b

**Comment by weppos** [comment]
Closed by #77
